x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_charts/risks_table.tsx

```diff
@@ -81,9 +84,25 @@ export const RisksTable = ({
   const sortedByComplianceScore = getTopRisks(cisSectionsEvaluations, maxItems);
 
   return (
-    <EuiFlexGroup direction="column" justifyContent="spaceBetween" gutterSize="none">
+    <EuiFlexGroup
+      direction="column"
+      justifyContent="spaceBetween"
+      gutterSize="none"
+      css={containerStyles}
+    >
       <EuiFlexItem>
         <EuiInMemoryTable<GroupedFindingsEvaluation>
+          className="risk-table"
+          css={
+            compact
+              ? css`
+                  margin-top: -10px;
+                  .euiTable .euiTableRow .euiTableRowCell {
+                    border-top: none;
+                  }
```

Comment:
This would be better to come from EUI as an optional property of the table, it's pretty hacky to do on our side. but anyway, consider naming that 10px, something like `topBorderHeight`, so we will understand what exactly this value compensates for.

Reply:
EUITable is pretty opinionated in table design.  From a product perspective, if CSP has its own unique UX then there will be customization.  Ideally, It's better to extend themes and handle customization in CSS versus adding component properties for certain table designs. Currently, we are handling CSS in `.tsx` files. 

`topBorderHeight` is a little confusing since the property `marginTop` doesn't `borderTop`.  I'll either rename margin-top:-10px or maybe add a comment to give more context.